crawler.py

```diff
@@ -0,0 +1,40 @@
+import requests
+import json
+
+url = "https://google.serper.dev/search"
+
+ruta = "api_share.txt"
+
+
+def leer_clave():
+    with open(ruta, 'r') as archivo:
+        return archivo.read().strip()
+
+
+def consultar_api():
+    payload = json.dumps({
+        "q": "site:sharegpt.com",
+        "num": 10
+    })
+
+    headers = {
+        'X-API-KEY': leer_clave(),
+        'Content-Type': 'application/json'
+    }
+
+    response = requests.request("POST", url, headers=headers, data=payload)
+
+    print(response.text)
```

Comment:
Esta función debería retornar la respuesta en un formato útil para que `limpiar_resp` pueda procesarlo.
```suggestion
    return response.json()
```